src/MobileStepper/MobileStepper.js

```diff
@@ -0,0 +1,165 @@
+// @flow weak
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { createStyleSheet } from 'jss-theme-reactor';
+import withStyles from '../styles/withStyles';
+import Paper from '../Paper';
+import Button from '../Button';
+import KeyboardArrowLeft from '../svg-icons/keyboard-arrow-left';
+import KeyboardArrowRight from '../svg-icons/keyboard-arrow-right';
+import { LinearProgress } from '../Progress';
+
+export const styleSheet = createStyleSheet('MuiMobileStepper', theme => ({
+  mobileStepper: {
```

Comment:
Will do - just an FYI tho, I was basing my work on the `AppBar` component which uses `appBar` not `root`.